package.json

```diff
@@ -6,9 +6,12 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.1",
+    "redux": "^4.0.5",
+    "typescript": "^4.1.3",
```

Comment:
- [ ] We still need to add `react-redux` package, as it will make it easier your react application to connect to the redux store.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone1/AddReduxAndReact-Redux.png)